src/codeu/chat/server/Server.java

```diff
@@ -221,20 +260,15 @@ public void run() {
     });
   }
 
-  public User newUser(String id, String name, String time){
-	  
-	  
-	  Uuid userid = new Uuid(Integer.parseInt(id));
+  //adds new user at the start
+  public void addNewUser(String id, String name, String time){
 	  
+	  //converts strings to necessary objects
+	  Uuid userid = new Uuid(Integer.parseInt(id));	  
 	  Time usercreation = Time.fromMs(Long.parseLong(time));

```

Comment:
I'm afraid there may be an input format exception with the Long.parseLong since time will be returned in HH:MM:SS.sss